addons/website_blog/controllers/main.py

```diff
@@ -215,9 +215,7 @@ def blog(self, blog=None, tag=None, page=1, search=None, **opt):
 
         if blog:
             values['main_object'] = blog
-            values['blog_url'] = QueryURL('', ['blog', 'tag'], blog=blog, tag=tag, date_begin=date_begin, date_end=date_end, search=search)
-        else:
-            values['blog_url'] = QueryURL('/blog', ['tag'], date_begin=date_begin, date_end=date_end, search=search)
+        values['blog_url'] = QueryURL('/blog', ['blog', 'tag'], blog=blog, tag=tag, date_begin=date_begin, date_end=date_end)
```

Comment:
Any reason to remove the `search` parameter? If yes, should not this be on its own commit with explanation?